packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -21,3 +21,13 @@ export const oidcClientMetadataGuard = z.object({
 });
 
 export type OidcClientMetadata = z.infer<typeof oidcClientMetadataGuard>;
+
+export const userLogPayloadGuard = z.object({
+  ip: z.string().optional(),
+  userAgent: z.string().optional(),
+  applicationId: z.string().optional(),
+  applicationName: z.string().optional(),
+  details: z.any().optional(), // To be read only by human
```

Comment:
should it be a string type?

Reply:
this should be json / object or something (if `zod` supports)

p.s. `To be read only by human` sounds ambiguous here. please specify.